apps/codeforafrica/src/lib/api.netlify-cms/sections/getError.js

```diff
@@ -0,0 +1,10 @@
+import getPages from "./getPages";
```

Comment:
Do we need this `getError` (and `getPages`)? Can't we reuse `getHeader` or `getHero`? Essentially that's all we're extracting from the 404/error page content.